migtests/scripts/run-test.sh

```diff
@@ -65,17 +65,22 @@ main() {
 	tail -20 ${EXPORT_DIR}/reports/report.txt
 
 	step "Export data."
-	export_data
-	ls -l ${EXPORT_DIR}/data
+	{ #TRY
+		export_data 
+		export_data_exit_code=$?
```

Comment:
Doesn't seem like we are using this variable anywhere?

Reply:
yes removed it.